packages/blade/src/components/Input/BaseInput/BaseInput.tsx

```diff
@@ -32,7 +32,7 @@ export type BaseInputProps = FormInputLabelProps &
     /**
      * Determines if it needs to be rendered as input or textarea
      */
-    as?: 'input' | 'textarea';
+    as?: 'input' | 'textarea' | 'button';
```

Comment:
The JSDoc comment might need an update now.

Reply:
done